### DISCUSSION TITLE
Buttons at top of page

### DISCUSSION THREAD

**Discussion by sarahmey** [forkcms/forkcms#3338] [Ideas / Feature Requests]
## Type

- Enhancement

## Problem description

At the pages, it would improve user-friendliness by having the publish / save draft / delete buttons also at the top of the page. This way you don't always have to scroll down to the buttons and it is in your view when you only have to make a small change at the top of the page.

## Steps to reproduce

Par example make a small change at the top of a page with a long template. The buttons aren't in your view so it's easy to forget to scroll to save. 

## Expected behavior / Proposed solutions

Put the buttons also on the top of the page

**Comment by carakas** [comment]
@sarahmey not sure if there is enough room for this

**Comment by Senjutsuu** [comment]
> @sarahmey not sure if there is enough room for this

We could go for fixed buttons on the bottom but not sure about the best approach for that.
